build.sh

```diff
@@ -19,8 +24,30 @@ fi
 __buildproj=$__scriptpath/build.proj
 __buildlog=$__scriptpath/msbuild.log
 
-# Run the restore build tools under xbuild to get NuGet and MSBuild pulled down
-xbuild "$__buildproj" /t:_RestoreBuildTools /nologo /verbosity:minimal "/fileloggerparameters:Verbosity=diag;LogFile=$__buildlog"
+# Pull NuGet.exe down if we don't have it already
+if [ ! -e "$__nugetpath" ]; then
+    which curl > /dev/null 2> /dev/null
+    if [ $? -ne 0 ]; then
```

Comment:
We should check for wget as well, it's very likely that at least one of the two is installed on almost all systems.